crates/re_arrow_store/tests/correctness.rs

```diff
@@ -493,3 +493,95 @@ pub fn init_logs() {
         re_log::setup_native_logging();
     }
 }
+
+// ---
+
+#[test]
+fn entity_min_time_correct() {
+    init_logs();
+
+    for config in re_arrow_store::test_util::all_configs() {
+        let mut store = DataStore::new(InstanceKey::name(), config.clone());
+        entity_min_time_correct_impl(&mut store);
+    }
+}
+
+fn entity_min_time_correct_impl(store: &mut DataStore) {
+    let ent_path = EntityPath::from("this/that");
+    let wrong_ent_path = EntityPath::from("this/that/other");
+
+    let positions = build_some_positions2d(3);
+    let colors = build_some_colors(3);
```

Comment:
@teh-cmc I think this doesn't respect your prescription of using only stuff from `re_log_types::example_components` or `re_types_core`, but it's done all over the place in this file already 🤷🏻

Reply:
Mistakes were made in the past, let's not repeat them :upside_down_face:

Reply:
Done 👍🏻